xbmc/pvr/windows/GUIWindowPVRGuide.cpp

```diff
@@ -46,14 +46,13 @@ CGUIWindowPVRGuide::CGUIWindowPVRGuide(bool bRadio) :
   m_refreshTimelineItemsThread(new CPVRRefreshTimelineItemsThread(this)),
   m_bRefreshTimelineItems(false)
 {
-  m_cachedTimeline = new CFileItemList;
+  m_cachedTimeline = std::make_shared<CFileItemList>();
```

Comment:
if this is the case, say hello to 14: https://github.com/xbmc/xbmc/search?utf8=%E2%9C%93&q=make_shared